### ISSUE TITLE
[FR] feature like database folder plugin but for tags

### ISSUE THREAD

**Issue by happyguy227** [vrtmrz/obsidian-tagfolder#29]
I know this FR is probably way out of the scope of this plugin, but its the 1 thing keeping me from switching entirely from folders to tags.

This plugin already brings the advantages of hierarchical navigation, but there are still some features that are only available to folders. One that's become really important to my workflow is the [database folder plugin](https://github.com/RafaelGB/obsidian-db-folder), which uses dataview to create a database of all the notes in a folder + edit/sort by different YAML metadata

If that could somehow be integrated into this plugin (but with 1 database table representing a tag rather than a folder), I think that would help bridge the gap between folders and tags.

**Comment by vrtmrz** [comment]
Thank you for your voice! This is the first time I have learned of `Database-folder`.
TagFolder is structuring tags as trees on demand, so I do not have a good idea to implement this yet. 
If I got inspiration, I will do this!

**Comment by happyguy227** [comment]
> Thank you for your voice! This is the first time I have learned of `Database-folder`. TagFolder is structuring tags as trees on demand, so I do not have a good idea to implement this yet. If I got inspiration, I will do this!

Yeah, I realize this plugin is more for creating a folder-like navigation for tags. I just thought I would open a FR here because it also would help structure/navigate tags and would ultimately have to integrate with this plugin somehow. Database folder creates a database of all notes in 1 folder using dataview. It probably has some dataview query in its code to get all those notes from 1 folder. I thought it might be possible to change that query to get all the notes from 1 tag instead, and then add a context menu option to create a database for a tag.
